ci/uc-data-load/jobs/preprod-uc-data-load.yml

```diff
@@ -26,6 +26,10 @@ jobs:
     - .: (( inject meta.plan.terraform-output-sdx ))
       params:
         TF_WORKSPACE: 'preprod'
+  - .: (( inject meta-uc-data-load.plan.clear-manifest ))
+    config:
+      params:
+        AWS_ACC: '((aws_account.preprod))'
```

Comment:
FYI you don't need inverted commas around the account variables, it works with or without

Reply:
Ah ok, will keep it in this one as it matches the rest of the pipeline, but won't use it for future ones I do.